static/styles.css

```diff
@@ -106,7 +113,7 @@ h5 {
 
 .consent-section {
 	overflow: hidden;
-	transition: height 0.5s;
+	transition: height 1s;
```

Comment:
if we want to update this to 1s we should probably update  consent1.html:39

Reply:
I changed this because for me the sliding in and out happens almost simultaneously making the content scroll up and then down very fast. Not a pleasant experience. I spent too much time trying to fix that, so I finally opted for making the transition slower. That actually made it a little easier on the eye.

Reply:
No problem -- The function `fix_heights` has to unset and reset this property - so if we change this here we need to change it there too I think.